app/styles/gavant-ember-table.scss

```diff
@@ -0,0 +1,220 @@
+@import 'gavant-ember-table/variables';
+@import 'ember-bootstrap/bootstrap';
```

Comment:
i don't think we want ember-bootstrap imported in the addon's styles, as our projects import bootstrap themselves (and usually directly via the main bootstrap repo). that way projects can use newer versions of bootstrap w/o us having to update this addon whenever a new version is released.

Reply:
In the gavant-ember-table.scss, we have `@include media-breakpoint-up()` which comes from bootstrap, so will this cause issues when running in other apps? Should I indicate that the application uses bootstrap in the readme?

Reply:
@napafundi it looks like that 1 instead of that `media-breakpoint-up` mixin usage is for the "static-table" style which removes the min-height that we had applied as part of the DRS sitewide ui review changes? If thats the case, I'd probably just remove that `static-table` style block entirely, as that's a more per-project type style/customization.

Otherwise, we'd probably just convert that mixin usage to a raw css media query breakpoint, so that we could shed the addon's dependency on bootstrap. looks like we prob can avoid that in this case though if the above is true.

Reply:
ahhh, yeah you're 100% correct.